rust/libmagic/src/file_types.rs

```diff
@@ -651,8 +651,8 @@ static FILE_TYPES: phf::Map<&'static str, FileTypeInfo> = phf_map! {
     "tbm" => FileTypeInfo { friendly_type: "Tableau Bookmark", mime_type: "application/octet-stream"},
     "tcap" => FileTypeInfo { friendly_type: "3rd Generation Partnership Project - Transaction Capabilities Application Part", mime_type: "application/vnd.3gpp2.tcap"},
     "tcl" => FileTypeInfo { friendly_type: "Tcl Script", mime_type: "application/x-tcl"},
-    "tds" => FileTypeInfo { friendly_type: "Tableau Data Source", mime_type: "application/octet-stream"},
-    "tdsx" => FileTypeInfo { friendly_type: "Tableau Data Source", mime_type: "application/octet-stream"},
+    "tds" => FileTypeInfo { friendly_type: "Tableau Data Source", mime_type: "application/tds"},
```

Comment:
Are these mime types commonly used? I don't see them listed in https://www.iana.org/assignments/media-types/media-types.xhtml

Reply:
I am not sure if they're commonly used. My understanding is that the mime-types used in here are being used by the summary code ([fetcher.rs#191](https://github.com/xetdata/xet-core/pull/304/files#diff-8bbee25fdd1cbb41082b2a710f90b0a3c5b4754fb0af2c4483f498e95b03c254R191-R200)) to select a type of summary. Using octet-stream is fairly broad and could result in us trying to generate tds summaries for many arbitrary binary types.

Reply:
Fair enough -- you're right that the logic there is affected by this mime type, so I agree let's use a specific type. LGTM.